### PR TITLE
Add FluxCD kustomize controller v0.24.4+update.1

### PR DESCRIPTION
**Pull request by chaudharyalka**
## What this PR does / why we need it
Add FluxCD kustomize controller v0.24.4+update.1

## Which issue(s) this PR fixes
Fixes: #[OLYMP-30817](https://jira.eng.vmware.com/browse/OLYMP-30817)

## Describe testing done for PR
Manually added package

### CODE REVIEW

**Review by seemiller — CHANGES_REQUESTED** [review]
@chaudharyalka It is not necessary to create a new `0.24.4+update.1` directory. You can update the vendir yaml files and sync the new package contents directly into the existing `0.24.4` directory. That will keep the directory structure much cleaner.

**Comment by chaudharyalka** [comment]
@seemiller As we discussed offline I have updated the PR. Please check.

File: addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ spec:
     spec:
       fetch:
         - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/fluxcd-kustomize-controller-bundle@sha256:424e08bc92768bbb179a8ba09c0062619dabcba1614736bbd69c025422bbd2f4
+            image: projects.registry.vmware.com/tce/fluxcd-kustomize-controller-bundle:0.24.4-update.1
```
Comment (seemiller) [inline addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml:14]: @chaudharyalka You don't want to use a tag here. On an OCI Registry, the tag can potentially be changed to another image/bundle, so there is no guarantee of what you're going to actually be referring to. By using the sha256, you're guaranteed to get the correct image.

Can you switch this back to using the sha?

**Review by seemiller — CHANGES_REQUESTED** [review]

File: addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml (hunk 1 above)
Comment (chaudharyalka) [reply addons/packages/fluxcd-kustomize-controller/0.24.4/package.yaml]: @seemiller Done.

**Pull request merged**